### PR TITLE
UserGuide.adoc: Standardise the import feature format and structure

### PR DESCRIPTION
**Pull request by Exeexe93**
Standardise the import feature format and structure in User Guide

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -143,20 +143,22 @@ Returns any person having names `Betsy`, `Tim`, or `John`
...
-
-You will be able to import orders via .csv files that are given to you by your company.
+==== Description
+You will be able to import orders via csv file that are given to you by your company.
 The CSV file will include all relevant and important details of the parcels; such as the customers’ names,
 transaction id of the parcels and the address of the customers, etc. +
 
-Format: `import FILE NAME`
+==== Format
+* `import FILE NAME`
```
Comment (khsc96) [inline docs/UserGuide.adoc]: Minor changes here `import` `FILE_NAME` to standardise.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -143,20 +143,22 @@ Returns any person having names `Betsy`, `Tim`, or `John`
...
 
 ****
-* Import the specific csv with the file name in data folder to the application
+* Import the specific csv with the file name in data folder to the application.
+* Only *one* csv file can be import at one time.
 * The FILE NAME should include the extension. For example: *orders.csv*.
 * The CSV file should store at data folder that is at the same directory as the JAR file.
```
Comment (khsc96) [inline docs/UserGuide.adoc]: Perhaps could include a more specific constraint here stating file can only be in .csv format

**Review by khsc96 — COMMENTED** [review]
Minor changes suggested, please do have a look!

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -143,20 +143,22 @@ Returns any person having names `Betsy`, `Tim`, or `John`
...
 
-Format: `import FILE NAME`
+==== Format
+* `import FILE NAME`
 
 ****
-* Import the specific csv with the file name in data folder to the application
+* Import the specific csv with the file name in data folder to the application.
+* Only *one* csv file can be import at one time.
 * The FILE NAME should include the extension. For example: *orders.csv*.
```
Comment (khsc96) [inline docs/UserGuide.adoc]: FILE_NAME here as well

Comment (Exeexe93) [reply docs/UserGuide.adoc]: Replace the FILE NAME with FILE_NAME

File: docs/UserGuide.adoc (hunk 1 above)
Comment (Exeexe93) [reply docs/UserGuide.adoc]: Resolved.

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -143,20 +143,23 @@ Returns any person having names `Betsy`, `Tim`, or `John`
...
 
-Format: `import FILE NAME`
+==== Format
+* `import FILE_NAME`
 
 ****
-* Import the specific csv with the file name in data folder to the application
-* The FILE NAME should include the extension. For example: *orders.csv*.
-* The CSV file should store at data folder that is at the same directory as the JAR file.
+* Only csv file could be import.
```
Comment (Cherweijie) [inline docs/UserGuide.adoc]: Should be 'imported'

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -143,20 +143,23 @@ Returns any person having names `Betsy`, `Tim`, or `John`
...
+* Only *one* csv file can be import at one time.
```
Comment (Cherweijie) [inline docs/UserGuide.adoc]: Should be 'imported'

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -143,20 +143,23 @@ Returns any person having names `Betsy`, `Tim`, or `John`
...
+* The FILE_NAME should include the extension. For example: *orders.csv*.
+* Import the specific csv with the file name in data folder to the application.
+* The csv file should store at data folder that is at the same directory as the JAR file.
```
Comment (Cherweijie) [inline docs/UserGuide.adoc]: I think this line has to be rephrased to be clearer

Comment (Cherweijie) [inline docs/UserGuide.adoc]: I think this line has to be rephrased to be clearer

**Review by khsc96 — CHANGES_REQUESTED** [review]
Some grammar issues mentioned by weijie, once resolved it's all good:)

File: docs/UserGuide.adoc (hunk 4 above)
Comment (Exeexe93) [reply docs/UserGuide.adoc]: Are you referring to line 155 or any other line?

File: docs/UserGuide.adoc (hunk 5 above)
Comment (Exeexe93) [reply docs/UserGuide.adoc]: Ok thanks! Changed!

File: docs/UserGuide.adoc (hunk 4 above)
Comment (Exeexe93) [reply docs/UserGuide.adoc]: Resolved

File: docs/UserGuide.adoc (hunk 6 above)
Comment (Exeexe93) [reply docs/UserGuide.adoc]: Change to `The folder, data, which the csv files are stored should be at the same directory as the JAR file.`

**Review by khsc96 — APPROVED** [review]
LGTM! Good job

**Review by khsc96 — APPROVED** [review]
LGTM

**Review by khsc96 — APPROVED** [review]
LGTM

**Review by Cherweijie — APPROVED** [review]
LGTM

**Pull request merged**